kyco/core/napps.py

```diff
@@ -7,7 +7,7 @@
 
 log = logging.getLogger(__name__)
 
-__all__ = ('KycoCoreNApp', 'KycoNApp')
+__all__ = ('KycoNApp')
```

Comment:
A valid tuple here would be `__all__ = ('KycoNApp',)` or simply `__all__ = 'KycoNApp',` (always with comma).